1228-public_page/1242-request_draft_doi_button_is_disabled_for_publication_with_existing_doi.feature

```diff
@@ -1,11 +1,10 @@
-Feature: Request/Draft Doi buttion is disabled for Publications with existing DOI
+Feature: Request/Draft DOI button is disabled for Publications with existing DOI

```

Comment:
Creator cannot request a DOI for a Publication that already has a DOI(?)

Reply:
I think it is relevant to mention both Request DOI (for a Published Publication) and Draft DOI (for a Draft Publication), since this story yields for both cases (but naturally only one at a time)